lib/_emerge/EbuildFetcher.py

```diff
@@ -373,7 +373,7 @@ def _async_uri_map(self):
         def cache_result(result):
             try:
                 self._uri_map = result.result()
-            except Exception:
+            except (CancelledError, Exception):
```

Comment:
Fixed to to use asyncio.CancelledError in e4f0fa4a316aba2cf2b1a47c9d2974710a1d8bd3.